support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -41,16 +33,19 @@
     @StepContextParameter
     /*package*/ transient Run run;
 
-    @StepContextParameter private transient TaskListener listener;
+    @StepContextParameter
+    private transient TaskListener listener;
 
-    @StepContextParameter private transient FlowNode node;
+    @StepContextParameter
+    private transient FlowNode node;
 
     /**
      * Result of the input.
      */
     private Outcome outcome;
 
-    @Inject(optional=true) InputStep input;
+    @Inject(optional=true)
+    InputStep input;
```

Comment:
In general, PRs should avoid making semantically null changes to lines not otherwise related to the diff. Whitespace-only changes make `git blame` less useful, increase the chance of conflicts during `git merge` or `git cherry-pick`, etc. And of course it is harder to review a longer diff.

In particular, you should configure your IDE to _not_ automatically reformat files just because you are editing something in them.